packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -570,17 +619,69 @@ export function markRootUpdated(root: FiberRoot, updateLane: Lane) {
 
   // Unsuspend any update at equal or lower priority.
   const higherPriorityLanes = updateLane - 1; // Turns 0b1000 into 0b0111
+  const equalOrLowerPriorityLanes = ~higherPriorityLanes;
+
+  // We use this below to mark the event time
+  const unsuspendedLanes = equalOrLowerPriorityLanes & root.suspendedLanes;
+
   root.suspendedLanes &= higherPriorityLanes;
   root.pingedLanes &= higherPriorityLanes;
+
+  if (eventTime !== -1) {
+    const eventTimes = root.eventTimes;
+    let lanes = updateLane | unsuspendedLanes;
+    while (lanes > 0) {
+      const index = ctrz(lanes);
+
+      // Assumes event times are monotonically increasing. Last one wins.
```

Comment:
Sorry, maybe this is me being dumb because it's early, but I'm not sure I understand this comment. Isn't the only assumption this code is making is that `eventTime` is positive? What does "last one wins" mean in this context?

Was this comment more applicable to old expiration times implementation?

Reply:
Oops, I meant "first one wins"

The idea is that the times always increase so there's no need to check if an existing time is earlier than the new time. The existing time will always be earlier.

Reply:
Haha, ok that makes a lot more sense. I even wondered if maybe that's what you meant but discarded it.